container-toolkit/cdi-support.md

```diff
@@ -77,7 +77,11 @@ However, the path to create and use can depend on the container engine that you
    ```
 
 ```{important}
-If you change the device or CUDA driver configuration, you must generate a new CDI specification.
+You must generate a new CDI specification after any of the following changes:
+
+- You change the device or CUDA driver configuration.
+- You upgrade the Linux kernel.
```

Comment:
There isn't anything kernel-specific in the CDI specification. It would be good to understand why this would be required?

Reply:
Glad you looked and clarified.  I'm OK with leaving this open for a few days and revising if we learn what the issue was.  If we never learn, then you or I can close it.  LMK if you prefer otherwise.

Reply:
@elezar, I replaced the advice based on a sentence you shared in the nvbug. PTAL [again](https://nvidia.github.io/cloud-native-docs/review/pr-83/container-toolkit/latest/cdi-support.html#procedure).

Suggs and corrections welcome, as always.

Reply:
Let's remove the kernel upgrade text. What about something along the lines of:

> You must generate a new CDI specification if the device or CUDA driver configuration changes. In addition, if a location such as `/var/run/cdi` -- which is cleared on boot -- is used, the CDI specifications must be regenerated at boot.

Reply:
Very sorry to have accidentally left the kernel upgrade mention.